src/bootstrap/src/core/build_steps/tool.rs

```diff
@@ -1186,8 +1186,18 @@ fn run_tool_build_step(
     }
 }
 
-tool_extended!(Cargofmt { path: "src/tools/rustfmt", tool_name: "cargo-fmt", stable: true });
-tool_extended!(CargoClippy { path: "src/tools/clippy", tool_name: "cargo-clippy", stable: true });
```

Comment:
Do we need to remove "cargo-fmt" and "cargo-clippy" from L1195 & L1214 below?

Reply:
Good point!